### DISCUSSION TITLE
How to backup user data and document

### DISCUSSION THREAD

**Discussion by mc-stephen** [appwrite/appwrite#3722] [Q&A] [answered]
Hello, so I created a user and some documents assign to that user ID, but I don't know how to back up my user and their document, just in case my VPS malfunction, and then being forced to buy a new one and install Appwrite again does that mean my user's details are gone like forever, if not so then how do I back it up to local storage and how will I upload it again.
Thanks

**Comment by stnguyen90** [comment, marked as answer]
For instructions on how to backup and restore, check out [this](https://youtube.com/watch?v=lM5yZEPtlvg) video.

**Reply by mc-stephen** [reply]
> For instructions on how to backup and restore, check out [this](https://youtube.com/watch?v=lM5yZEPtlvg) video.

hmmm, just saw the video, looks good but the explanation does not do for me, tho I can do my research.
thanks for showing me this, now I know where to look. 
tho if you have more knowledge you can share with me on this I will very much love that.

**Reply by stnguyen90** [reply]
What's missing from the video? Also, if you're satisfied, would you mark my response as the answer?

**Reply by mc-stephen** [reply]
nothing is missing at least for now, but I will like it if you guys can add a feature where we can just back up our data with a click of a button, thanks.

also, I don't know how to mark your response as the answer, where should I click?

**Reply by stnguyen90** [reply]
@Emeka212, right below my answer there should be a button that says "Mark as answer":

<img width="487" alt="image" src="https://user-images.githubusercontent.com/1477010/192633539-a2630ad0-e1d7-4dc6-a1d5-0b48ddc66a16.png">

**Reply by mc-stephen** [reply]
lmao, i have seen that before just didn't know why i couldn't find it again, thanks for letting me know
